crates/bevy_sprite/src/render/sprite.frag

```diff
@@ -20,5 +20,7 @@ void main() {
         sampler2D(ColorMaterial_texture, ColorMaterial_texture_sampler),
         v_Uv);
 # endif
+    if(color.a < 0.5) discard;
```

Comment:
I believe I read somewhere that `discard` hurts performance.

Reply:
This is hardware dependent ([source](https://stackoverflow.com/a/8509158)). In any case, the performance hit shouldn't be noticeable.

Reply:
Why a cutout value of 0.5? Isn't that high?

Reply:
That's the "standard" default value from my experience. I'm looking into making it customisable, but we'll see if time permits.